.github/workflows/build_test_deploy.yml

```diff
@@ -629,6 +629,9 @@ jobs:
       - run: xvfb-run node run-tests.js --timings -g ${{ matrix.group }}/18
         if: ${{needs.build.outputs.docsChange != 'docs only change'}}
 
+      - run: NEXT_TEST_MODE=start PKG_MANAGER=pnpm xvfb-run node run-tests.js test/e2e/styled-jsx/index.test.ts
```

Comment:
This doesn't seem ideal, I think the more ideal way to go about this is to run 2 tests with different inputs. I'll update `test/e2e/styled-jsx/index.test.ts` to reflect that.

Reply:
Updated to this approach: https://github.com/vercel/next.js/pull/36585/files#diff-e1b72b64ea6f1f2833924b8ce31283c68e6df60bd064fc4a312a968b82f79839R42